jobs/admin.py

```diff
@@ -1,6 +1,6 @@
 from django.contrib import admin
 
-from .models import Company, Job
+from .models import Company, Job, Meetup
 
 
 def make_published(modeladmin, request, queryset):
```

Comment:
using "job" will work, but it's a bit confusing when this method is to be used for both jobs and meetups.  Is there a better name?  Maybe just item, or obj?